infra/src/main/resources/exception/exceptionInfo.json

```diff
@@ -202,5 +202,35 @@
     "id": 35,
     "reason": "Shard Operator received an invalid URL for the BridgeIngress endpoint.",
     "type": "PLATFORM"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.platform.ProvisioningFailureException",
+    "id": 36,
+    "reason": "An unspecified error occurred handling your request.",
+    "type": "PLATFORM"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.platform.ProvisioningTimeOutException",
+    "id": 37,
+    "reason": "Your request was not complete within the configured time-out.",
+    "type": "PLATFORM"
+  },
```

Comment:
Is this a `PLATFORM` exception? Should it be `USER`? Same for the others

Reply:
Well.. these exceptions come about from the PLATFORM failing to do something the USER requested.

I thought USER [exceptions](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/infra/src/main/java/com/redhat/service/smartevents/infra/exceptions/definitions/user/ExternalUserException.java#L6) were limited to errors relating to the User's input?